tools/check_jsonnet

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+success=1
+JSONNET_FILES=$(git ls-files data/ | grep '.json\?net$' | xargs echo)
+LIBSONNET_FILES=$(git ls-files data/ | grep '.libson\?net$' | xargs echo)
```

Comment:
ahm ok, I got it, you need single regex to combine both extensions.

Reply:
egrep is not supported, so any suggestion to combine them together?

Reply:
`grep -E` ?

Reply:
yes, grep -E works, updated. Thanks.